ts/nni_manager/common/training_service_v3.ts

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ *  TODO
+ *  This type is planned to be an interface.
+ *  To minimize the change, it is currently identical to TrialJobMetric.data
+ **/
+export type Metric = string;
+
+/**
+ *  TODO
+ *  This type is planned to be an interface.
+ *  To minimize the change, it is currently identical to HyperParameters.value
+ **/
+export type Parameter = string;
+
+export interface EnvironmentInfo {
+    id: string;
+    // TODO
+}
+
+export interface TrainingServiceV3 {
```

Comment:
Hold on... where does this come from?

Reply:
Well this PR bases on v3 dev branch and was supposed to be merged after #5211 . I manually removed some files later but seems no that clean.
The merge schedule is as following:
 1. bump version (this)
 2. http command channel (#5216)
 3. gpu scheduler (#5219)
 4. trial keeper (#5229)
 5. local training service (next)  ⬅  this one depends on the old dev branch #5211